docs/hub/_toctree.yml

```diff
@@ -182,6 +182,8 @@
       title: Custom Python Spaces
     - local: spaces-add-to-arxiv
       title: How to Add a Space to ArXiv
+    - local: spaces-secrets-scanner
+      title: Secrets Scanner
```

Comment:
would maybe put it more under security, i.e. here:

<img width="363" alt="image" src="https://github.com/huggingface/hub-docs/assets/326577/2c48af04-5f3b-4d03-b815-cdb4abaf98b3">